src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -25,8 +25,11 @@ struct segment_info_t {
   segment_seq_t journal_segment_seq = NULL_SEG_SEQ;
 
 
+  bool out_of_line;
```

Comment:
Add an initializer or force usage of a constructor.

Reply:
done:-)